variables.tf

```diff
@@ -335,3 +336,27 @@ variable "custom_node_policy_arns" {
   default     = []
   description = "Custom node policy arns"
 }
+
+variable "enable_ssm_patching" {
+  type        = bool
+  default     = false
+  description = "Enables Systems Manager to patch nodes"
+}
+
+variable "ssm_iam_patching_policy" {
+  type        = string
+  default     = "cms-cloud-ssm-iam-policy-v3"
+  description = "SSM IAM policy for patching"
+}
+
+variable "patch_group" {
+  type        = string
+  default     = "AL2"
+  description = "SSM Patching group for instances"
+}
+
+variable "patch_window" {
```

Comment:
let's rename these variables to clarify "ssm_tag_patch_group", "ssm_tag_patch_window" and include in the description a link to the docs: https://cloud.cms.gov/patching-prerequisites